android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/MeasureExtensions.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.util.extension
+
+import org.apache.commons.lang3.StringUtils
+import org.hl7.fhir.r4.model.MeasureReport
+import org.opencds.cqf.cql.evaluator.measure.common.MeasurePopulationType
+
+val MONTH_NAMES =
+  listOf("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec")
```

Comment:
Can we have a followup PRs for migrating these hard coded month strings to `string.xml` for purposes of translation.

Reply:
https://github.com/opensrp/fhircore/issues/1689